boards.txt

```diff
@@ -50604,3 +50604,194 @@ kodedot.recipe.hooks.objcopy.postobjcopy.3.pattern_args=
 kodedot.recipe.output.save_file={build.project_name}.ino.bin
 
 ##############################################################
+
+# FED4 Board
+fed4.name=FED4
+fed4.vid.0=0x303A
+fed4.pid.0=0x82E5
+fed4.upload_port.0.vid=0x303A
+fed4.upload_port.0.pid=0x82E5
+fed4.bootloader.tool=esptool_py
+fed4.bootloader.tool.default=esptool_py
+
+fed4.upload.tool=esptool_py
+fed4.upload.tool.default=esptool_py
+fed4.upload.tool.network=esp_ota
+
+fed4.upload.maximum_size=1310720
+fed4.upload.maximum_data_size=327680
+fed4.upload.flags=
+fed4.upload.extra_flags=
+fed4.upload.use_1200bps_touch=false
+fed4.upload.wait_for_upload_port=false
+
+fed4.serial.disableDTR=false
+fed4.serial.disableRTS=false
+
+fed4.build.tarch=xtensa
+fed4.build.bootloader_addr=0x0
+fed4.build.target=esp32s3
+fed4.build.mcu=esp32s3
+fed4.build.core=esp32
+fed4.build.variant=fed4
+fed4.build.board=FED4
+
+fed4.build.usb_mode=1
+fed4.build.cdc_on_boot=0
+fed4.build.msc_on_boot=0
+fed4.build.dfu_on_boot=0
+fed4.build.f_cpu=240000000L
+fed4.build.flash_size=16MB
+fed4.build.flash_freq=80m
+fed4.build.flash_mode=dio
+fed4.build.boot=qio
+fed4.build.boot_freq=80m
+fed4.build.partitions=default_16MB
+fed4.build.defines=
+fed4.build.loop_core=
+fed4.build.event_core=
+fed4.build.psram_type=qspi
+fed4.build.memory_type={build.boot}_{build.psram_type}
+
+## IDE 2.0 Seems to not update the value
+fed4.menu.JTAGAdapter.default=Disabled
+fed4.menu.JTAGAdapter.default.build.copy_jtag_files=0
+fed4.menu.JTAGAdapter.builtin=Integrated USB JTAG
+fed4.menu.JTAGAdapter.builtin.build.openocdscript=fed4-builtin.cfg
+fed4.menu.JTAGAdapter.builtin.build.copy_jtag_files=1
+fed4.menu.JTAGAdapter.external=FTDI Adapter
+fed4.menu.JTAGAdapter.external.build.openocdscript=fed4-ftdi.cfg
+fed4.menu.JTAGAdapter.external.build.copy_jtag_files=1
+fed4.menu.JTAGAdapter.bridge=ESP USB Bridge
+fed4.menu.JTAGAdapter.bridge.build.openocdscript=fed4-bridge.cfg
+fed4.menu.JTAGAdapter.bridge.build.copy_jtag_files=1
+
+fed4.menu.FlashMode.qio=QIO 80MHz
+fed4.menu.FlashMode.qio.build.flash_mode=dio
+fed4.menu.FlashMode.qio.build.boot=qio
+fed4.menu.FlashMode.qio.build.boot_freq=80m
+fed4.menu.FlashMode.qio.build.flash_freq=80m
+fed4.menu.FlashMode.qio120=QIO 120MHz
+fed4.menu.FlashMode.qio120.build.flash_mode=dio
+fed4.menu.FlashMode.qio120.build.boot=qio
+fed4.menu.FlashMode.qio120.build.boot_freq=120m
+fed4.menu.FlashMode.qio120.build.flash_freq=80m
+fed4.menu.FlashMode.dio=DIO 80MHz
+fed4.menu.FlashMode.dio.build.flash_mode=dio
+fed4.menu.FlashMode.dio.build.boot=dio
+fed4.menu.FlashMode.dio.build.boot_freq=80m
+fed4.menu.FlashMode.dio.build.flash_freq=80m
+fed4.menu.FlashMode.opi=OPI 80MHz
+fed4.menu.FlashMode.opi.build.flash_mode=dout
+fed4.menu.FlashMode.opi.build.boot=opi
+fed4.menu.FlashMode.opi.build.boot_freq=80m
+fed4.menu.FlashMode.opi.build.flash_freq=80m
```

Comment:
@strid3r21 Also OPI flash mode should be removed, depending the module used.

Reply:
removed